ui/packages/docs/src/components/Params/character_data.json

```diff
@@ -809,5 +809,27 @@
       "param": "hold_ticks",
       "desc": "Number of ticks for Hold E. Default 6, maximum 6. Only works if hold = 1."
     }
+  ],
+  "travelerhydro": [
+    {
+      "ability": "skill",
+      "param": "hold",
+      "desc": "0 for Tap (default), 1 for Hold."
+    },
+    {
+      "ability": "skill",
+      "param": "hold_ticks",
+      "desc": "Number of ticks for Hold E. Default 22, maximum 22. Only works if hold = 1."
```

Comment:
Please change this to `Number of ticks for Hold E. Default 22, maximum 22, minimum 0. Only works if hold = 1. If 0, then a shorter version of Tap will be performed.`